towhee/runtime/data_queue.py

```diff
@@ -113,14 +143,22 @@ def col_size(self) -> int:
 
     def clear_and_seal(self):
```

Comment:
in which case do we need this method?

Reply:
when node failed, clear all data and stop running

Reply:
When failed, clear all datas and exit